language/types_values_variables.md

```diff
@@ -142,16 +146,17 @@ may appear more than once in the hierarchy (e.g. a `Scalar` is both `Any` and `D
        |  |     |- (Integer with range inside another Integer)
        |  |  |- Float[from, to]
        |  |     |- (Float with range inside another Float)
+       |  |  |- Timespan
+       |  |  |- Timestamp
        |  |
        |  |- String[from, to]
        |  |  |- Enum[*strings]
        |  |  |- Pattern[*patterns]
        |  |
        |  |- Boolean
        |  |- Regexp[pattern_string]
-       |
-       |- SemVer
-       |- SemVerRange
+       |  |- SemVer
+       |  |- SemVerRange
```

Comment:
Regexp is problematic. My mistake.  Since we have that problem, and the constraints of Data are not really used for what it was intended for, I don't think it matters much if SemVer and SemVerRange are of Scalar type.
The problem is that Regexp, SemVer, etc. cannot be trivially represented as data directly represented in JSon (you would need to add interpretation).